espree.js

```diff
@@ -3412,18 +3519,29 @@ function parseVariableIdentifier() {
 }
 
 function parseVariableDeclaration(kind) {
-    var init = null, id,
-        marker = markerCreate();
-
-    id = parseVariableIdentifier();
-
-    // 12.2.1
-    if (strict && syntax.isRestrictedWord(id.name)) {
```

Comment:
need to check what this was doing. replaced with current `harmony`, but it may have had some value...